config/install/acquia_lift.settings.yml

```diff
@@ -17,4 +17,5 @@ thumbnail: []
 visibility:
   # Track, only when the page URL falls OUTSIDE OF these path patterns.
   path_patterns: "/admin\n/admin/*\n/batch\n/node/add*\n/node/*/*\n/user/*/*\n/block/*"
+content_replacement_mode: 'untrusted'
```

Comment:
I actually think replacement is the better approach as my observations with users is the opposite. Deployment carries with it a connotation of a heavy action and is somewhat technical - I'm "deploying" something or something is being "deployed." Most end-users I've talked to and tested very much understand the concept that the rules they are creating replace things. As such, my recommendation would be let's go with replacement.